target/linux/mediatek/dts/mt7986a-ruijie-rg-x60-pro-ubootmod.dts

```diff
@@ -0,0 +1,28 @@
+// SPDX-License-Identifier: (GPL-2.0 OR MIT)
+
+/dts-v1/;
+#include "mt7986a-ruijie-rg-x60-pro.dtsi"
+
+/ {
+	compatible = "ruijie,rg-x60-pro-ubootmod", "mediatek,mt7986a";
+	model = "Ruijie RG-X60 Pro (OpenWrt U-Boot layout)";
+};
+
+&chosen {
+	rootdisk = <&ubi_rootdisk>;
+	bootargs = " root=/dev/fit0 rootwait";
```

Comment:
It's better to move this bootargs into the bootloader. Ref: https://git.openwrt.org/?p=openwrt%2Fopenwrt.git&a=search&h=HEAD&st=grep&s=root%3D%2Fdev%2Ffit0+rootwait